Makefile

```diff
@@ -156,6 +168,13 @@ prerelease: clean fmtcheck test cross ## Pre-release target to verify tests pass
 		exit 1; \
 	fi
 
+.PHONY: systemtray
+systemtray: fmtcheck test
```

Comment:
I do not think we need to do `fmtcheck test` as there is separate target for these. I do not see any benefit for making it implicit.